src/geometry/CTiglBSplineFit.h

```diff
@@ -21,9 +21,8 @@
 
 #include <vector>
 #include <TColStd_Array1OfReal.hxx>
-#include <Handle_Geom_BSplineCurve.hxx>
-
-class TColgp_Array1OfPnt;
+#include <Geom_BSplineCurve.hxx>
+#include <TColgp_Array1OfPnt.hxx>
```

Comment:
Any reason, why we can't forward declare the class ```TColgp_Array1OfPnt``` anymore?

Reply:
yes, TColgp_Array1OfPnt used to be a type in older OCC versions and is a typedef in newer OCC versions.

in my TColgp_Array1OfPnt.hxx:
```
typedef NCollection_Array1<gp_Pnt> TColgp_Array1OfPnt;
```

unfortunately, typedefs cannot be forward declared, so you would have to repeat the typedef, requiring to include NCollection_Array1.hxx. And switching this between older/newer OCC versions with the preprocessor looks horrible.

Reply:
Good thought but does not help here, since the unit tests don't run with OpenCASCADE 7.2.0. I think, the change is correct, but I would not hurt to test it properly. Can you do it? I don't have OCCT 7.2.0 right now and I am sitting in the train and cannot install it.

Reply:
Okay, in this case we should let it like it is.